app/assets/stylesheets/delete_target.scss

```diff
@@ -16,3 +21,19 @@
   border-color: black;
   border-radius: 10px;
 }
+
+.button-delete {
+  text-align: center;
+  margin: 10px 50px 0px 50px;
+  background-color: black;
+  border: none;
+  padding: 16px 32px;
+  -webkit-transition-duration: 0.4s; /* Safari */
```

Comment:
What is this for?

Reply:
The transition duration property is to determine the speed of the "hover" effect.